ports/stm32/Makefile

```diff
@@ -403,8 +403,10 @@ $(BUILD)/firmware.elf: $(OBJ)
 	$(Q)$(LD) $(LDFLAGS) -o $@ $^ $(LIBS)
 	$(Q)$(SIZE) $@
 
-PLLVALUES = boards/pllvalues.py
-MAKE_PINS = boards/make-pins.py
+PLLVALUES_SRC = boards/pllvalues.py
+PLLVALUES = $(PYTHON) $(PLLVALUES_SRC)
+MAKE_PINS_SRC = boards/make-pins.py
+MAKE_PINS = $(PYTHON) $(MAKE_PINS_SRC)
```

Comment:
I think for these scripts, they are such an integral part of the build system and are assumed to be Python scripts that it's not worth doubling the number of makefile-variables in this way.  I don't think there are any sensible scenarios where these values are changed by the user.  Only `$(PYTHON)` would change and that's already taken care of by using `$(PYTHON)` in the makefile rule.  So I'd say these should stay as they were.